webapp/src/store/sidebar.ts

```diff
@@ -68,6 +68,9 @@ const sidebarSlice = createSlice({
                         updateAt: updatedCategory.updateAt,
                     }
                 }
+
+                // sort categories alphabetically
+                state.categoryAttributes.sort((a, b) => a.name.localeCompare(b.name))
             })
         },
```

Comment:
Should only sort once.
```suggestion
            })
            // sort categories alphabetically
            state.categoryAttributes.sort((a, b) => a.name.localeCompare(b.name))
        },
```

Reply:
Done.